token.go

```diff
@@ -71,9 +71,5 @@ func getToken(port int) (string, error) {
 
 	token := <-done
 
-	err := srv.Close()
-	if err != nil {
-		return token, errors.Wrap(err, "Error shutting down server")
-	}
```

Comment:
Logic for deletion is that, anyway program will exit and server will be closed or I miss something here.